frontend/packages/console-shared/src/constants/common.ts

```diff
@@ -71,3 +71,5 @@ export enum FLAGS {
   CONSOLE_EXTERNAL_LOG_LINK = 'CONSOLE_EXTERNAL_LOG_LINK',
   CONSOLE_YAML_SAMPLE = 'CONSOLE_YAML_SAMPLE',
 }
+
+export const BREAKPOINTLG = 1800;
```

Comment:
This name is confusing because it no longer matches the Boostrap/Sass breakpoint `lg`. I would give this a specific name like `NOTIFICATION_DRAWER_BREAKPOINT` since it's only used for the drawer.